rule/if_return.go

```diff
@@ -44,7 +44,7 @@ func (w *lintElseError) Visit(node ast.Node) ast.Visitor {
 				continue
 			}
 			assign, ok := s.Init.(*ast.AssignStmt)
-			if !ok || len(assign.Lhs) != 1 || !(assign.Tok == token.DEFINE || assign.Tok == token.ASSIGN) {
+			if !ok || len(assign.Lhs) != 1 || assign.Tok != token.DEFINE && assign.Tok != token.ASSIGN {
```

Comment:
I'm not sure if this became more readable. Let's revert.

Reply:
I agree

This one is implied by QF0001

Please take a look here, on how to disable the rule that implied this.

https://github.com/gofr-dev/gofr/blob/dac6bdc735c3b60ccfb17d8e4a799c23e3d7383d/.golangci.yml#L141

I had to do it for gofr too

- https://github.com/gofr-dev/gofr/pull/1708